src/main/java/org/broadinstitute/hellbender/tools/walkers/UnmarkDuplicates.java

```diff
@@ -17,15 +17,44 @@
 import java.util.Collections;
 import java.util.List;
 
-@DocumentedFeature
+/**
+ * Removes duplicate read tags.
```

Comment:
I think you mean --> Unflags the 0x400 duplicate SAM flag.

'Tag' is used to describe the elements that come after the base qualities, e.g. AS tag.